specifyweb/frontend/js_src/lib/components/Security/Institution.tsx

```diff
@@ -99,7 +99,7 @@ function InstitutionView({
                 ) : (
                   commonText.loading()
                 )}
-                <div className="flex gap-2">
+                <div className="flex flex-wrap gap-2">
```

Comment:
If you have the desire, going though all the places where flexbox is used and checking if it makes sense to add `flex-wrap` would be a good thing